homeassistant/components/zwave_js/api.py

```diff
@@ -2611,3 +2613,19 @@ async def websocket_invoke_cc_api(
             msg[ID],
             result,
         )
+
+
+@callback
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/get_global_settings",
```

Comment:
I'd think that "global" is unclear what it means. I'd call it eg "get_integration_settings".